src/app/pages/LandingPage.svelte

```diff
@@ -186,10 +212,36 @@
   </header>
 
   <section class="explanation" id="explanation">
-    <!-- TODO: figure that explains the flow -->
+    <div class="explanation__workflow">
+      <img
+        src="assets/workflow-spreadsheet.svg"
+        alt=""
+        role="presentation"
+        class="explanation__workflow__image" />
+
+      <div class="explanation__workflow__transform">
+        Predictive Text Studio
+        <img
+          class="explanation__workflow__transform__arrow"
+          src="assets/workflow-arrow.svg"
+          alt=""
+          role="presentation" />
+      </div>
+
+      <img
+        src="assets/workflow-keyboard.svg"
+        alt=""
+        role="presentation"
+        class="explanation__workflow__image" />
+    </div>
```

Comment:
Since this "explanation" is _content_ (it's briefly explaining what the app does) and not just decorative, I'd add `alt=` attributes and put everything in a [`<figure>` element]():

```suggestion
    <figure class="explanation__workflow">
      <img
        src="assets/workflow-spreadsheet.svg"
        alt="a spreadsheet of words in your language"
        class="explanation__workflow__image" />

      <div class="explanation__workflow__transform">
        Predictive Text Studio
        <img
          class="explanation__workflow__transform__arrow"
          src="assets/workflow-arrow.svg"
          alt=""
          role="presentation" />
      </div>

      <img
        src="assets/workflow-keyboard.svg"
        alt="your words appear in the predictive text banner on your phone's keyboard"
        class="explanation__workflow__image" />
    </figure>
```